gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```
@@ -2411,4 +2433,9 @@ private boolean requesterShouldPay(String bucketName) {
   public AtomicLong getObjectStatistics(GoogleCloudStorageStatistics key) {
     return objectStatistics.get(key);
   }
+
+  @Override
+  public AtomicLong getHttpStatistics(GoogleCloudStorageStatistics key) {
+    return httpStatistics.get(key);
```

Please add a null check here

---

Updated

---

httpStatistics can still be null, we dont want to throw the exception here

---

Updated